README.md

```diff
@@ -86,19 +86,19 @@ Add the call to `inc()` in your code. Try experiment with the placement of where
 
 ---
 
-So far, we've been able to instrument our application, such that it is now exporting metrics about its runtime behavior. However, we still need to collect those metrics and store the data in a way that we can query it back out, in order to graph it over time and make dashboards.
+So far we've been able to instrument our application, such that it is now exporting metrics about its runtime behavior. However, we still need to collect those metrics and store the data in a way so we can query it back out in order to graph it over time and make dashboards.
 
-There is a `prometheus.yaml` configuration file here in the repo, which is already set up to scrape metrics from our application. We can run both our application, Prometheus, and Grafana inside Docker, so that they are easily able to find each other.
+There is a `prometheus.yaml` configuration file here in the repo, which is already set up to scrape metrics from our application. We will run our application, Prometheus, and Grafana inside Docker, so that they are easily able to find each other.
 
 #### Run the application, Prometheus and Grafana in Docker
 
-To build the application Docker image, and start the application container, Prometheus and Grafana together, run the following command (from the root of this repo):
+To build the application Docker image and start the application container as well as Prometheus and Grafana together, run the following command (from the root of this repo):
 
 ``` sh
 docker-compose up --build
 ```
 
-*If you see errors it may be because you still have the previous version of the application running, and using the same port as you are now trying to access with Docker.*
+*If you see errors it may be because you still have the previous version of the application running and might be using the same port as you are now trying to access with Docker.*
```

Comment:
```suggestion
*If you see errors it may be because you still have the previous version of the application running and therefore might be using the same port as you are now trying to access with Docker.*
```